# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/quota/ActorsParent.cpp
# Commit: a604c144191f
# Full Hash: a604c144191f5c3c65169c072d4e52a1e2c1c23f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2024-10-09 09:46:37
# Regressor Bug: 1866402
# File Overlap Count: 1
# Description:
#   Bug 1866402 - Change nsIQuotaManagerService::ClearStoragesForPrincipal to support clearing of entire origins only; r=dom-storage-reviewers,jari
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D195505
# ==============================================================================

diff -r 281a926003fd -r a604c144191f dom/quota/ActorsParent.cpp
--- a/dom/quota/ActorsParent.cpp	Wed Oct 09 05:34:36 2024 +0000
+++ b/dom/quota/ActorsParent.cpp	Wed Oct 09 05:34:37 2024 +0000
@@ -5834,13 +5834,11 @@
 
 RefPtr<BoolPromise> QuotaManager::ClearStoragesForOrigin(
     const Maybe<PersistenceType>& aPersistenceType,
-    const PrincipalInfo& aPrincipalInfo,
-    const Maybe<Client::Type>& aClientType) {
+    const PrincipalInfo& aPrincipalInfo) {
   AssertIsOnOwningThread();
 
-  auto clearOriginOp =
-      CreateClearOriginOp(WrapMovingNotNullUnchecked(this), aPersistenceType,
-                          aPrincipalInfo, aClientType);
+  auto clearOriginOp = CreateClearOriginOp(WrapMovingNotNullUnchecked(this),
+                                           aPersistenceType, aPrincipalInfo);
 
   RegisterNormalOriginOp(*clearOriginOp);
 